newsletters/templates/newsletters/create.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'tgrsite/main.html' %}
+{% block title %}
+New Newsletter
+{% endblock %}
+{% block body %}
+
+<h1>New Newsletter</h1>
+<form action="" method="POST">
+	{% csrf_token %}
+	{{ form.non_field.errors }}
+	{% for field in form %}
+		<div>
+			<div>
+				{% if field.field.required %} * {% endif %}
+				{{field.label_tag}}
+			</div>
+			<div>
+				{{ field }}
+			</div>
+		</div><br>
+	{% endfor %}
+	<input type="submit" class="form-control">
+</form>
+<a href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet">Markdown</a> tentatively supported in the body.
```

Comment:
Might not be necessary since simpleMDE is going to be implemented soon

Reply:
Fixed.